nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
```

Comment:
Perhaps I am not fully in line with that due to not fully understanding it, but if it's only used for "display" I am ok with putting it back in. Please confirm